### PR TITLE
Feature/add delete shortcut

### PR DESCRIPTION
**Pull request by k-y0u**
Add management of the delete shortcut to delete components and links.

### CODE REVIEW

File: src/draw/DefaultDrawer.js [hunk 1]
```diff
@@ -1471,6 +1482,32 @@ class DefaultDrawer {
...
+      type: 'Drawer',
+      action: 'delete',
+      status: 'success',
+      components: [this.actions.selection.current.id],
+    });
+
+    this.draw(this.rootId);
+  }
+
+  removeLinkHandler() {
```
Comment (justinehell) [inline src/draw/DefaultDrawer.js:1498]: No doc

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/draw/DefaultDrawer.js [hunk 2]
```diff
@@ -550,6 +550,17 @@ class DefaultDrawer {
     this.setViewPortAction();
 
     this.pluginData.emitEvent({ id, status: 'success' });
+
+    d3.select('body')
+      .on('keyup', (event) => {
+        if (event.key === 'Delete' && this.actions.selection.current) {
+          if (this.actions.selection.current.__class === 'Component') {
+            this.removeComponentHandler();
+          } else if (this.actions.selection.current.__class === 'Link') {
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/draw/DefaultDrawer.js:559]: Of course, it's possible but I don't think it's really useful. @Zorin95670, any opinion ?

File: src/draw/DefaultDrawer.js [hunk 3]
```diff
@@ -1471,6 +1482,32 @@ class DefaultDrawer {
     }
   }
 
+  removeComponentHandler() {
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/draw/DefaultDrawer.js:1489]: Done

File: src/draw/DefaultDrawer.js (hunk 1 above)
Comment (k-y0u) [reply src/draw/DefaultDrawer.js:1506]: Done

**Review by DimitriChauvel — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Review by justinehell — APPROVED** [review]

**Review by DimitriChauvel — APPROVED** [review]

**Pull request merged**